src/main/java/org/sagebionetworks/bridge/services/AccountService.java

```diff
@@ -94,7 +84,7 @@ protected String generateGUID() {
      * Search for all accounts across apps that have the same Synapse user ID in common, 
      * and return a list of the app IDs where these accounts are found.
      */
-    public List<String> getAppIdsForUser(String synapseUserId) {
+    public List<String> getAppIdsForUser2(String synapseUserId) {
```

Comment:
Just curious what this rename is for? Are you differentiating from the same name method in AdminAccountService?

Reply:
Actually should this method just be deleted?

Reply:
Yes this was a step to ensuring all code was pointed at the same method in AdminAccountService; I will delete it.